Team6220_PowerPlay/src/main/java/org/firstinspires/ftc/team6220_PowerPlay/AutoFramework.java

```diff
@@ -0,0 +1,99 @@
+package org.firstinspires.ftc.team6220_PowerPlay;
+
+
+abstract public class AutoFramework extends BaseAutonomous {
+
+    /**
+     * @param AutoSelector Assign to 0 or 1, corresponding to BlueLeft/RedLeft, or BlueRight/RedRight respectively
+     * @throws InterruptedException If you haven't added a config for that autonomous
+     */
+    public void runAuto(int AutoSelector) throws InterruptedException {
+
+
+        //set values here
+        int driveCourse;
+        int targetDistance = 11;
+        int[] signalArray;
+        switch (AutoSelector) {
+
+            //BlueLeft & RedLeft
+            case 0:
+                signalArray = new int[] {90, 33, 90, 11, -90, 11};
+                driveCourse = -90;
+                break;
+            //BlueRight & RedRight
+            case 1:
+                signalArray = new int[] {90, 11, -90, 11, -90, 33};
+                driveCourse = 90;
+                break;
+            default:
+                throw new IllegalArgumentException("you gotta edit AutoFramework to do that");
```

Comment:
I think this error message should be clearer. Maybe say you need to add a new case to AutoFramework.